weaver/core/drivers/fabric-driver/server/satp.ts

```diff
@@ -0,0 +1,363 @@
+import satp_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_pb';
+import satp_grpc_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_grpc_pb';
+import driverPb from '@hyperledger/cacti-weaver-protos-js/driver/driver_pb';
+import logger from './logger';
+import { credentials } from '@grpc/grpc-js';
+import { SatpAssetManager, HashFunctions } from '@hyperledger/cacti-weaver-sdk-fabric'
+import fs from 'fs';
+import path from 'path';
+import { Gateway, Network, Wallets, Contract, X509Identity } from 'fabric-network'
+import { getNetworkGateway } from "./fabric-code";
+import { getDriverKeyCert } from './walletSetup';
+
+const DB_NAME: string = "driverdb";
+const DRIVER_ERROR_CONSTANTS = JSON.parse(
+    fs.readFileSync(
+        path.resolve(__dirname, '../constants/driver-error-constants.json'),
+    ).toString(),
+);
+
+const getWallet = (walletPath: string) => {
+    return Wallets.newFileSystemWallet(walletPath);
+};
+
+async function performLockHelper(
+    performLockRequest: driverPb.PerformLockRequest,
+    networkName: string
+): Promise<any> {
```

Comment:
Don't use any (I think this function doesn't return anything at the moment)

Reply:
Agreed, I support replacing it with `unknown`

Reply:
@outsidethecode Please fix this as per Peter's suggestion.

Reply:
Fixed.